openmrs/omod/src/main/java/org/openmrs/projectbuendia/webservices/rest/PatientResource.java

```diff
@@ -369,38 +367,18 @@ private static void setLocation(Patient patient, String locationUuid) {
     }
 
     private SimpleObject searchInner(RequestContext requestContext) throws ResponseException {
-        List<Patient> patients = new ArrayList<>();
+        List<Patient> patients;
         String patientId = requestContext.getParameter("id");  // single patient by ID; overrides
-        // "q"
         if (patientId != null) {
             List<PatientIdentifierType> idTypes = new ArrayList<>();
             idTypes.add(DbUtil.getMsfIdentifierType());
             patients = patientService.getPatients(null, patientId, idTypes, true);
         } else {
-            String nameQuery = requestContext.getParameter("q");
-            patients = filterPatients(nameQuery, patientService.getAllPatients());
+            patients = patientService.getAllPatients();
```

Comment:
Any particular reason why this deletion is part of this PR?  It's true that the client doesn't use this at the moment, just wondering if it could be a separate change.